engine/install/ubuntu.md

```diff
@@ -104,6 +104,15 @@ Docker from the repository.
       "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
       sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
     ```
+    
+    > **Note**
+    > In case your Linux distro is based on Ubuntu (e.g Linux Min 21.1), use `"$UBUNTU_CODENAME"` instead of `$VERSION_CODENAME"`:
+    > ```console
+    > $ echo \
+    >   "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] {{ download-url-base }} \
+    >   "$(. /etc/os-release && echo "$UBUNTU_CODENAME")" stable" | \
+    >   sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    > ```
```

Comment:
O, I only saw your comment now, after reviewing 😅 While we don't support derivatives, having the note contextually where the failure might otherwise occur would be more helpful. I've suggested a condensed version of the note which hopefully doesnt create too much noise.

Reply:
Agree!. I added a new commit based on your suggestion @dvdksn